client/src/services/resourceService.test.js

```diff
@@ -129,4 +129,42 @@ describe("ResourceService", () => {
 			await expect(service.suggest({})).rejects.toThrow("something went wrong");
 		});
 	});
+
+	describe("getAccountResources", () => {
+		it("includes query parameters if supplied", async () => {
+			let request;
+			server.use(
+				rest.get("/api/resources", (req, res, ctx) => {
+					request = req;
+					return res(ctx.json({ resources: [] }));
+				})
+			);
+			await service.getAccountResources({ page: 23, perPage: 56 });
+			expect(request.url.searchParams.get("account")).toBe("true");
+			expect(request.url.searchParams.get("page")).toBe("23");
+			expect(request.url.searchParams.get("perPage")).toBe("56");
+		});
+
+		it("resolves with user resources if request succeeds", async () => {
+			server.use(
+				rest.get("/api/resources", (req, res, ctx) =>
+					res(ctx.json({ resources: [] }))
+				)
+			);
+			await expect(
+				service.getAccountResources({ account: true, page: 1, perPage: 5 })
+			).resolves.toEqual({ resources: [] });
+		});
+
+		it("returns user resources on success", async () => {
+			await server.use(
+				rest.get("/api/resources", async (req, res, ctx) => {
+					return res(ctx.json({ resources: [] }));
+				})
+			);
+			await expect(
+				service.getAccountResources({ account: true, page: 1, perPage: 5 })
+			).resolves.toEqual({ resources: [] });
+		});
+	});
```

Comment:
❌ this exact same test appears twice with different names (and slightly different arrow function formatting) - and in neither case does the response actually include any resources.